plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -1203,40 +1181,40 @@ def main():
 
         # Ensure source VM name exists when cloning
         if -1 == vmid:
-            module.fail_json(msg='VM with name = %s does not exist in cluster' % clone)
+            module.fail_json(vmid=vmid, msg='VM with name = %s does not exist in cluster' % clone)
```

Comment:
The documentation only says that `vmid` should be returned on success. I guess it makes sense to return it on failure if you have it, but in this case `vmid` is not available.